tests/e2e/env/setup.sh

```diff
@@ -136,6 +136,10 @@ cli wp option set woocommerce_currency "USD"
 cli wp option set woocommerce_product_type "both"
 cli wp option set woocommerce_allow_tracking "no"
 
+echo "Creating customer user..."
+cli wp user delete customer --yes
+cli wp user create customer customer@woocommercecoree2etestsuite.com --role=customer --user_pass=password
```

Comment:
As I understand the customer is required for saved cards test. Can we create it in spec or tests setup file (`e2e/setup/jest-setup.js`) to reuse data from `config/test.json`?

Reply:
Yup! Good idea. I've implemented it in https://github.com/Automattic/woocommerce-payments/pull/816/commits/3c3d0518d1bf2283e84dadb69e2127f333d15b81. We still need to figure out a way to make it work when running the tests interactively though.